### PR TITLE
[improvement](iceberg)Parallelize splits for count(*)

### PR DESCRIPTION
**Pull request by wuwenchi**
## Proposed changes

1. Parallelize splits to prevent bottlenecks in a single split.
2. Only a single column needs to be resized.

### CODE REVIEW

**Comment by wuwenchi** [comment]
run buildall

**Comment by wuwenchi** [comment]
run buildall

**Comment by wuwenchi** [comment]
run buildall

File: be/src/vec/exec/format/table/iceberg_reader.cpp [hunk 1]
```diff
@@ -95,6 +94,9 @@ IcebergTableReader::IcebergTableReader(std::unique_ptr<GenericReader> file_forma
             ADD_CHILD_TIMER(_profile, "DeleteFileReadTime", iceberg_profile);
     _iceberg_profile.delete_rows_sort_time =
             ADD_CHILD_TIMER(_profile, "DeleteRowsSortTime", iceberg_profile);
+    if (range.table_format_params.iceberg_params.__isset.row_count) {
+        _remaining_push_down_count = range.table_format_params.iceberg_params.row_count;
```
Comment (morningman) [inline be/src/vec/exec/format/table/iceberg_reader.cpp:98]: the `_remaining_push_down_count` is not initialized if `row_count` is not set.

**Comment by wuwenchi** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by hubgeter — APPROVED** [review]
LGTM

**Pull request merged**